### ISSUE TITLE
Not running properly

### ISSUE THREAD

**Issue by behnamprime** [Heumi/BEGAN-tensorflow#6]
Hello, When I run this code with:
python3 main.py -f 1 -p "began" -trd "celeba" -tro "crop" -trs 64  -z 128 -em 128 -fn 64  -b 16 -lr 1e-4 -gm 0.4 -g "0" 
or any other configurations, it works till shuffle done and it stops!
any suggestion? did anyone else got same error?

**Comment by s-zk** [comment]
Create the folder "128_crop" under directory ./Data/celeba, then run ./Data/celeba/face_detect.py

Look at line 23 in file ./Data/celeba/face_detect.py, 

                cv2.imwrite('128_crop/' + fn[:-4] + '_crop' + fn[-4:], image_resize)

This line means to save file under directory ./Data/celeba/128_crop, but if "128_crop" does not exist, there are no warnings, no errors, no exceptions. So create the folder and run "face_detect.py", you can see the cropped files generated in path "./Data/celeba/face_detect.py". It works in my case.

Good luck.

**Comment by jiyuay** [comment]
@s-zk thank you so much! Have you figure out how to use the pretrained model?

**Comment by Phajuer** [comment]
> Hello, When I run this code with:
> python3 main.py -f 1 -p "began" -trd "celeba" -tro "crop" -trs 64 -z 128 -em 128 -fn 64 -b 16 -lr 1e-4 -gm 0.4 -g "0"
> or any other configurations, it works till shuffle done and it stops!
> any suggestion? did anyone else got same error?

hi！Did you figure that out,i'm haveing the same problem
